src/pages/Refund.tsx

```diff
@@ -116,19 +116,23 @@ const Refund = () => {
                         undefined,
             )
             .map(async (swap) => {
-                const res = await getSwapStatus(swap.asset, swap.id);
-                if (
-                    !updateSwapStatus(swap.id, res.status) &&
-                    Object.values(swapStatusFailed).includes(res.status)
-                ) {
-                    if (res.status !== swapStatusFailed.SwapExpired) {
+                try {
+                    const res = await getSwapStatus(swap.asset, swap.id);
+                    if (
+                        !updateSwapStatus(swap.id, res.status) &&
+                        Object.values(swapStatusFailed).includes(res.status)
+                    ) {
+                        if (res.status !== swapStatusFailed.SwapExpired) {
+                            addToRefundableSwaps(swap);
+                            return;
+                        }
+
+                        // Make sure coins were locked for the swap with status "swap.expired"
+                        await getSubmarineTransaction(swap.asset, swap.id);
                         addToRefundableSwaps(swap);
-                        return;
                     }
-
-                    // Make sure coins were locked for the swap with status "swap.expired"
-                    await getSubmarineTransaction(swap.asset, swap.id);
-                    addToRefundableSwaps(swap);
+                } catch (_) {
+                    // do nothing, there is already a request error in the console
```

Comment:
An error of a failed HTTP request, if that happens to be enabled in your browser
Add a warning log

Reply:
done